biorxiv/corpora_comparison/figure_generator.py

```diff
@@ -0,0 +1,409 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Figures for Corpora Comparison between bioRxiv,  Pubmed Central, New York Times
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+import numpy as np
+import pandas as pd
+
+from cairosvg import svg2png
+from IPython.display import Image
+import plotnine as p9
+
+from annorxiver_modules.corpora_comparison_helper import(
+    calculate_confidence_intervals,
+    create_lemma_count_df,
+    plot_pointplot,
+    plot_bargraph,
+    plot_point_bar_figure
+)
+
+
+# In[2]:
+
+
+subset = 20
+
+
+# # KL Divergence Graph
+
+# In[3]:
+
+
+kl_divergence_df = pd.read_csv(
+    "output/comparison_stats/corpora_kl_divergence.tsv", 
+    sep="\t"
+)
+kl_divergence_df.head()
+
+
+# In[4]:
+
+
+g = (
+    p9.ggplot(
+        kl_divergence_df.replace({
+        "biorxiv_vs_nytac":"biorxiv_vs_reference",
+        "pmc_nytac": "pmc_vs_reference"
+        })
+    )
+    + p9.aes(x="factor(num_terms)", y="KL_divergence", fill="comparison")
+    + p9.geom_col(stat='identity', position='dodge')
+    + p9.scale_fill_manual(["#b2df8a", "#1f78b4", "#a6cee3"])
+    + p9.labs(
+        x = "Number of terms evaluated",
+        y = "KL Divergence"
+    )
+    + p9.theme_seaborn(context='paper')
+)
+g.save("output/figures/corpora_kl_divergence.png", dpi=500)
+print(g)
+
+
+# # bioRxiv vs Pubmed Central
+
+# In[5]:
+
+
+full_text_comparison = pd.read_csv(
+    "output/comparison_stats/biorxiv_vs_pmc_comparison.tsv",
+    sep="\t"
+)
+full_text_comparison.head()
+
+
+# ## Line Plots
+
+# In[6]:
+
+
+full_plot_df = calculate_confidence_intervals(
+    full_text_comparison
+)
+full_plot_df.head()
+
+
+# In[7]:
+
+
+plot_df = (
+    full_plot_df
+    .sort_values("odds_ratio", ascending=False)
+    .head(subset)
+    .append(
+        full_plot_df
+        .sort_values("odds_ratio", ascending=False)
+        .iloc[:-2]
+        .tail(subset)
+    )
+    .replace('rna', 'RNA')
+)
+plot_df.head()
+
+
+# In[8]:
+
+
+g = plot_pointplot(plot_df, "bioRxiv vs PMC Odds Ratio")
```

Comment:
Does it make sense that neurons and genome are at a high odds compared to PMC?

Reply:
> Does it make sense that neurons and genome are at a high odds compared to PMC?

Yes. As the greatest difference between bioRxiv and PMC are topic specific terms related to neuroscience and bioinformatics. PMC contains those topics, but has a wider range of documents and textual content.